android/src/main/java/com/emeraldsanto/encryptedstorage/RNEncryptedStorageModule.java

```diff
@@ -7,36 +7,53 @@
 import androidx.security.crypto.MasterKey;
 import com.facebook.react.bridge.Promise;
 import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContext;
 import com.facebook.react.bridge.ReactContextBaseJavaModule;
 import com.facebook.react.bridge.ReactMethod;
 
+import com.facebook.react.bridge.ReadableMap;
+
 public class RNEncryptedStorageModule extends ReactContextBaseJavaModule {
 
     private static final String NATIVE_MODULE_NAME = "RNEncryptedStorage";
-    private static final String SHARED_PREFERENCES_FILENAME = "RN_ENCRYPTED_STORAGE_SHARED_PREF";
 
     private SharedPreferences sharedPreferences;
+    private MasterKey masterKey;
+
+    private String getStorageName(ReadableMap options) {
+      String bundleId = this.getReactApplicationContext().getPackageName();
+      String storageName = options.hasKey("storageName") ?
```

Comment:
You are not setting any default value for `options`, nor checking if is it defined.

This line throws if called without an `options`, for example:

```
EncryptedStorage.getItem("myValue")
```